ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -5,14 +5,11 @@ import { getBlockExplorerLink } from '@metamask/etherscan-link';
 import { TransactionType } from '@metamask/transaction-controller';
 import SenderToRecipient from '../../ui/sender-to-recipient';
 import { DEFAULT_VARIANT } from '../../ui/sender-to-recipient/sender-to-recipient.constants';
-import Disclosure from '../../ui/disclosure';
-import TransactionActivityLog from '../transaction-activity-log';
```

Comment:
The `TransactionActivityLog` does appear in another location of the application I'm assuming we just want to remove it from this instance @chaoticgoodpanda?